modules/data-entry/src/main/frontend/src/questionnaire/VocabularyQuestion.jsx

```diff
@@ -27,67 +27,41 @@ import Question from "./Question";
 import QuestionnaireStyle from "./QuestionnaireStyle";
 
 import AnswerComponentManager from "./AnswerComponentManager";
-import VocabularySelector from "../vocabSelector/select.jsx";
+import MultipleChoice from "./MultipleChoice";
+import VocabularyQuery from "../vocabQuery/query.jsx";
 
 // Component that renders a vocabulary question.
-// Selected answers are placed in a series of <input type="hidden"> tags for
-// submission.
-//
-// Required arguments:
-//  text: String containing the question to ask
-//  sourceVocabularies: Array denoting the vocabularies sources
-//
-// Optional arguments:
-//  maxAnswers: Integer indicating the maximum number of terms allowed
-//
-// Other arguments are passed to the VocabularySelector component
 //
 // Sample usage:
 //
 // <VocabularyQuestion
-//   text="Does the patient have any co-morbidities (non cancerous), including abnormal incidental imaging/labs?"
-//   sourceVocabularies={["hpo"]}
-//   />
-// <VocabularyQuestion
-//   text="Does the patient have any skin conditions?"
-//   sourceVocabularies={["hpo"]}
-//   vocabularyFilter={["HP:0000951"]}
-//   />
-// <!-- Alternate method of specifying arguments -->
-// <VocabularyQuestion
-//   questionDefintion={{
+//   questionDefinition={{
 //     text: "Does the patient have any skin conditions?",
-//     sourceVocabularies={["hpo"]},
-//     vocabularyFilter: ["HP:0000951"]
+//     sourceVocabularies=["HP"],
```

Comment:
Oh, so many errors in here...

```suggestion
//     sourceVocabularies: ["HP"],
```